models/public/colorization-siggraph/colorization-siggraph.md

```diff
@@ -0,0 +1,158 @@
+
+## Accuracy
+
+The accuracy metrics calculated on ImageNet
+validation dataset using [VGG16](https://arxiv.org/abs/1409.1556) caffe
+model and colorization as preprocessing.
+
+For preprocessing `rgb -> gray -> coloriaztion` recieved values:
+
+| Metric         | Value with preprocessing   | Value without preprocessing |
+|----------------|----------------------------|-----------------------------|
+| Accuracy top-1 |                     58.25% |                      70.96% |
+| Accuracy top-5 |                     81.78% |                      89.88% |
+
+## Performance
+
+## Input
+
+### Original model
+
+1. Image, name - `data_l`,  shape - `1,1,256,256`, format is `B,C,H,W` where:
+
+   - `B` - batch size
+   - `C` - channel
+   - `H` - height
+   - `W` - width
+
+   Channel order is L-channel.
+
+2. Image, name - `user_ab`,  shape - `1,2,256,256`, format is `B,C,H,W` where:
+
+   - `B` - batch size
+   - `C` - channel
+   - `H` - height
+   - `W` - width
+
+   Channel order is AB channels of LAB-image. Input for user points.
+
+3. Mask, name - `user_map`,  shape - `1,1,256,256`, format is `B,C,H,W` where:
+
+   - `B` - batch size
+   - `C` - number of flags for pixel
+   - `H` - height
+   - `W` - width
+
+   This input is  a binary mask indicating which points are
+   provided by the user. The mask differentiates unspecified points
+   from user-specified gray points with (a,b) = 0.
+   If point(pixel) was specified the flag will be equal to 1.
+
+### Converted model
+
+1. Image, name - `data_l`,  shape - `1,1,256,256`, format is `B,C,H,W` where:
+
+   - `B` - batch size
+   - `C` - channel
+   - `H` - height
+   - `W` - width
+
+   Channel order is L-channel.
+
+2. Image, name - `user_ab`,  shape - `1,2,256,256`, format is `B,C,H,W` where:
+
+   - `B` - batch size
+   - `C` - channel
+   - `H` - height
+   - `W` - width
+
+   Channel order is AB channels of LAB-image. Input for user points.
+
+3. Mask, name - `user_map`,  shape - `1,1,256,256`, format is `B,C,H,W` where:
+
+   - `B` - batch size
+   - `C` - number of flags for pixel
+   - `H` - height
+   - `W` - width
+
+   This input is  a binary mask indicating which points are
+   provided by the user. The mask differentiates unspecified points
+   from user-specified gray points with (a,b) = 0.
+   If point(pixel) was specified the flag will be equal to 1.
+
+> **NOTE**: You don't need to specify all 3 inputs to use the model. If you dont't want to use local user hints (user points), you can use only `data_l` input.
+
+## Output
+
+### Original model
+
+Image, name - `color_ab`,  shape - `1,2,256,256`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is AB channels of LAB-image.
+
+### Converted model
+
+Image, name - `color_ab`,  shape - `1,2,256,256`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is AB channels of LAB-image.
```

Comment:
model inputs and output are the same before and after conversion. Probably you should specify them only once without separation on original and converted model (it will be enough) or need to say in converted model related section that nothing changed